### DISCUSSION TITLE
Broader Temperature Range Values for other markets.

### DISCUSSION THREAD

**Discussion by actionpotato** [revk/xESP32-Faikout#152] [Ideas]
I just got my first Faikin installed, and it is fantastic. I have one running on a FTXS18WVJU9. It has integrated with Home Assistant Perfectly. (Still need to install the other four)

Please note, I am in the United States, so my Daikin Models are probably very different from the UK.

One concern I have, is that the lowest possible Temperature I can set for heating seems to be 16 C  or ~61 F. And the Highest possible value I can set is 32 C  ~90 F.

<img width="835" alt="Screenshot 2023-11-22 at 12 26 56 PM" src="https://github.com/revk/ESP32-Faikin/assets/30300219/79a1f835-1204-4989-a1b5-692f23f3ffb5">

<img width="483" alt="Screenshot 2023-11-22 at 12 31 51 PM" src="https://github.com/revk/ESP32-Faikin/assets/30300219/64798edf-437b-41f1-a11e-3d2668773b37">

<img width="467" alt="Screenshot 2023-11-21 at 6 05 48 PM" src="https://github.com/revk/ESP32-Faikin/assets/30300219/2d2a2826-f03d-4fe9-b3ed-a338410ed1ce">

In Home Assistant I obviously get the same values. However, when using the Official Daikin Wifi Modules BRP072A43 I get a minimum heat value of  45 F ~7 C, and a maximum of 95 F ~35 C.

<img width="400" alt="Screenshot 2023-11-22 at 12 28 14 PM" src="https://github.com/revk/ESP32-Faikin/assets/30300219/5353c3f1-7bd4-428a-a390-c1c85952a79d">

<img width="399" alt="Screenshot 2023-11-22 at 12 28 31 PM" src="https://github.com/revk/ESP32-Faikin/assets/30300219/e1948312-d6a2-4fae-816b-3456e848a936">

Is it possible to give the Faikin the same range for heating? 60 F is too warm for the garage we have in the winter, we only need around 50-55 F.  I am not sure the best or safest way to accomplish this - maybe a flag for other markets in settings?

When googling this problem, it seems that the EU versions of the Daikin systems go from 16 C to 32 C just as you have coded into the system (which is totally logical): https://www.daikinac.com/content/assets/DOC/Leaflet%20-%20BRC1E71%20-%20Daikin%20AC.pdf

These temp ranges of 45-95 F seem to match the IR remotes that came with my units as well.  I am guessing the US systems are different in the total temp range?

Or perhaps am I just missing something in the directions for the Temp range and have mine set incorrectly?

**Comment by revk** [comment]
I may be able to allow it to try a wide range - I'll look in the morning.

**Comment by revk** [comment]
Sorry for the delay.

As I thought, we already handle this. There are settings which can be set via MQTT for min and max temperatures (in Celsius).

Do you have MQTT set up?

![image](https://github.com/revk/ESP32-Faikin/assets/996983/682d09f6-b070-40d0-97ec-b12bf5ef73aa)

**Reply by actionpotato** [reply]
Ah, I figured you might! Yes, I have MQTT set up, and I am using Home Assistant with the Faikin.

**Reply by revk** [reply]
OK sending `setting/nameoffaikin/tmin` with a payload of the min (Celsius) value, integer, will set the min. Same with `tmax` for maximum. That should do what you need.

**Comment by actionpotato** [comment]
Awesome, thanks so much. Hope you get lots of sales on these great little units!

**Reply by revk** [reply]
Thanks, tell your friends :-)
